### DISCUSSION TITLE
Welcome to open-tv Discussions!

### DISCUSSION THREAD

**Discussion by Fredolx** [Fredolx/open-tv#65] [Announcements]
Come discuss Open TV here!

**Comment by ke0eyh** [comment]
I just discovered Open-TV. This application ROCKS! Very fast and light I'm no coder but of all the IPTV apps I've tried on Linux, Open-TV is the best hands down! Thank you for making and maintaining this application. ****I💯

**Comment by normsky001** [comment]
Also just discovered your app Open-TV.....i downloaded onto my laptop via microsoft store.....i made a payment of 2.49 sterling and wanted to check if you are able to benefit from my subscription or do microsoft keep the payment.....kind regards Normsky

**Reply by Fredolx** [reply]
Hi, it goes to me. Thanks for your donation.

**Comment by Fikasz** [comment]
This app is awesome. Definitely something that Linux needed :D 
Keep up the good work. Awaiting updates with great anticipation.

**Comment by ginofalaci** [comment]
Is there any chance to have an Apple TV release?
